app/templates/screen/Presenter.kt.mustache

```diff
@@ -18,7 +18,7 @@ class {{screenUpperCamel}}Presenter(private val context: Context) {
     }
 
     private lateinit var subscriptions: CompositeSubscription
-    private var view: {{screenUpperCamel}}ViewContract? = null
```

Comment:
There is no reason for ViewContracts to be nullable. They are never set back to null.